### ISSUE TITLE
MoMA Postcard / Add white background as default for stamps when designing

### ISSUE THREAD

**Issue by bencebitmark** [bitmark-inc/autonomy-apps#2962]

**Comment by phuocbitmark** [comment]
@bencebitmark  in our stamp design table, we have 100 cells, if it has no border by default, when we draw a cell, we remove the border. so if we add white color as default, that means there would be no border, or do you want to alway keep the border

**Comment by phuocbitmark** [comment]
![Screenshot_20230927-140000_Autonomy (Dev)](https://github.com/bitmark-inc/autonomy-apps/assets/110799144/40957b66-720b-484c-8d6b-277444d5fbb7)

**Comment by phuocbitmark** [comment]
@bencebitmark we can also draw the border only if it white. i think that would look nice

**Comment by bencebitmark** [comment]
> we can also draw the border only if it white. i think that would look nice
@ppupha yeah agree, i was thinking this too. only show border if it's white.

so also on the default state, show the grid too.

**Comment by longbmk** [comment]
@phuocbitmark The Stamp Posstcard button is still enabled although user click back or clear stamp color. It should check at least 1 cell has color for all buttons.
https://github.com/bitmark-inc/autonomy-apps/assets/109057362/670c830d-f4a0-4689-b137-3c8bb361051d

**Comment by phuocbitmark** [comment]
@longbmk i think we allow it to have all white stamp when we add white pallet, a lot of artists took advantage of it to have a white background and just use the signature pen to draw thing.
we engineer just want to make sure users do not keep skipping and do nothing, if they want a white background, we should respect that.

**Comment by longbmk** [comment]
> @longbmk i think we allow it to have all white stamp when we add white pallet, a lot of artists took advantage of it to have a white background and just use the signature pen to draw thing.
> we engineer just want to make sure users do not keep skipping and do nothing, if they want a white background, we should respect that.

Are you ok with this to ship @bencebitmark

**Comment by bencebitmark** [comment]
yeah, agree. if at least one cell is coloured, even if white, let the user continue. 

if they colour one, then undo, we'll still need to require them to colour at least one cell – even if white – to continue.
